src/main/java/com/exactpro/th2/mstore/MessageStore.java

```diff
@@ -51,21 +58,28 @@ public static void main(String[] args) {
             shutdownManager.registerResource(cradleManager);
             CradleStorage storage = cradleManager.getStorage();
 
+            ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor(
+                    new ThreadFactoryBuilder().setNameFormat("error-collector-%d").build());
```

Comment:
Probably, you should use THREAD_FACTORY static field from ErrorCollector here

Reply:
We probably should remove the factory from `ErrorCollector` class if we have decided to move it to `MessageStore` class